GenderPayGap.WebUI/Views/Errors/UserNotRegisteredToReportForOrganisation.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@page
+@using GenderPayGap.Core
+@model GenderPayGap.WebUI.ErrorHandling.UserNotRegisteredToReportForOrganisationException
+
+@{
+    ViewBag.Title = "Permission Denied - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-xl">
+            You don't have permission to report on behalf of this organisation
+        </h1>
+
+        @{
+            // TODO: Link this to the add organisation journey (once it's been built!)
+        }

```

Comment:
This is now ready.

It can be something like:
```cshtml
@if (FeatureFlagHelper.IsFeatureEnabled(FeatureFlag.NewAddOrganisationJourney)) 
{ 
    <p class="govuk-body">
        <a href="@Url.Action("ChooseSector", "AddOrganisationChooseSector")"
           class="govuk-link">Add organisation</a> 
    </p> 
} 
else 
{ 
    <p class="govuk-body">
        <a href="@Url.Action(nameof(RegisterController.OrganisationType), "Register")"
           class="govuk-link">Register an organisation</a> 
    </p> 
} 
```